web/src/components/PoiSharedChildren.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactElement } from 'react'
```

Comment:
Not really happy with the naming `PoiSharedChildren`, other suggestions?

Reply:
Not really 🤔